src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestSeverity.java

```diff
@@ -0,0 +1,13 @@
+/*xml
+<module name="Checker">
+  <module name="LineLength">
```

Comment:
to make it work add

```
        moduleMappings.put("LineLength",
                "com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck");
```

to InlineConfigParser.java

Reply:
added this as you suggested, now test case is passing but when I manually mutate the code and run it again it is still passing, this should not be happening, it should be failing... I have pushed the changes, can you inspect this pls?

the mutated code is as followed:

**before mutation:**
```java
public final void setSeverity(String severity) {
    this.severity = SeverityLevel.getInstance(severity);
}
```

**after mutation:**
```java
public final void setSeverity(String severity) {
}
```

Reply:
I have unresolved this conversation... waiting for your reply @romani

Reply:
Uhh, can we reuse code that use xml config load from examples execution?
We do not need special code to parse , it is regular xml config, we already have code to load it properly.

Reply:
Do you mean by the new `createConfiguration()` method I created? It is there to check for the global properties and add it to the config. The old method was not checking for the global properties, I tried to modify the old method to check for the global properties but then other inputs examples tests started failing so I had to create the new method for that.

Reply:
Do you mean that we should use the old verifyWithInlineConfigParser() method & createConfiguration() method instead of the new one I made? I had to make them to add the global properties to the config, I've explained the reason here: https://github.com/checkstyle/checkstyle/pull/14462#issuecomment-1958094593 @romani

Reply:
do you mean by the new `createConfiguration()` method I created? if yes then here is the reason explaining that: https://github.com/checkstyle/checkstyle/pull/14462#issuecomment-1962315124 , I also created the `verifyWithInlineConfigParserGlobalProperites()` before but then I tried modifying the the current `verifyWithInlineConfigParser()` to use the new createConfiguration() and everything was working fine. @romani